README.md

```diff
@@ -103,6 +103,12 @@ packed into the `plugin_instance` dimension using the following syntax:
 ```
 plugin_instance:value[dim1=value1,dim2=value2,...]
 ```
+The dimensions can then be used as filters. Using the same examples as above, such filters would look like:
```

Comment:
Ah, gotcha gotcha. That does make sense. I agree then that it probably does not belong here, but that information does need to go somewhere. Any recommendations?